pyomo/scripting/plugins/solve.py

```diff
@@ -29,6 +29,10 @@ def create_parser(parser=None):
         action='store',
         dest='solver',
         default=None)
+    parser.add_argument('--solver-manager',
+        action='store',
+        dest='solver_manager',
+        default='serial')
```

Comment:
This makes me nervous: it implies users have more options available to them than they do.  What happens if the user specifies something like `async`?

Does this also represent a change in the way users are supposed to interact with NEOS from the command line?  Is there documentation that needs to be updated?

Reply:
Ahh - after discussing this with @whart222, this is only promoting an option that was previously available through the specific solver interface up to the subcommand level, and does not represent a change in the user interface.